qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg

```diff
@@ -121,15 +123,15 @@
                     read_rng_cmd = "WIN_UTILS:\\random_%PROCESSOR_ARCHITECTURE%.exe"
                     driver_test_names = 'rng'
                 - @with_storage:
-                   images += " stg0"
-                   image_name_stg0 = "images/stg0"
-                   image_size_stg0 = 5G
-                   remove_image_stg0 = yes
-                   force_create_image_stg0 = yes
-                   driver_test_names = 'iozone'
-                   driver_test_params = {'images': 'stg0'}
-                   iozone_cmd_opitons = " -azR -r 64k -n 512M -g 1G -M -I -i 0 -i 1 -b iozone.xls -f %s:\testfile"
-                   variants:
+                    images += " stg0"
+                    image_name_stg0 = "images/stg0"
+                    image_size_stg0 = 5G
+                    remove_image_stg0 = yes
+                    force_create_image_stg0 = yes
+                    driver_test_names = 'iozone'
+                    driver_test_params = {'images': 'stg0'}
+                    iozone_cmd_opitons = " -azR -r 64k -n 512M -g 1G -M -I -i 0 -i 1 -b iozone.xls -f %s:\testfile"
+                    variants:
```

Comment:
Seems these code change was by mistake?

Reply:
NO. The original config lack one space, therefore CI check failed.

Reply:
Ok.